docs/edgeql/statements/describe.rst

```diff
@@ -0,0 +1,197 @@
+.. _ref_eql_statements_describe:
+
+DESCRIBE
+========
+
+:eql-statement:
+
+``DESCRIBE`` -- provide human-readable description of a schema or a
+schema object
+
+.. eql:synopsis::
+
+    DESCRIBE SCHEMA [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
```

Comment:
```suggestion
    DESCRIBE SCHEMA [ AS {DDL | SDL | TEXT | TEXT VERBOSE} ];
```

^ afaict VERBOSE only does something in conjunction with TEXT

Reply:
Yes, but it's valid syntax with everything. We might want to add some comments in verbose mode of DDL and SDL in the future. So I wouldn't alter the grammar just yet.

Reply:
OK, I guess I'll change the synopsis without altering the grammar.

Reply:
>  OK, I guess I'll change the synopsis without altering the grammar.

yes.

> So I wouldn't alter the grammar just yet.

Yes, I'd do that in a follow up PR :)  Changing grammar is easy.  The fact that `DESCRIBE SCHEMA AS DDL VERBOSE` does nothing but is accepted is a confusing bug.  **When** you make it work we can re-enable the syntax in the grammar.

**In short: let's please have no active syntax that doesn't work and is there just because it might work in the future.**

Reply:
OK, I'll make a follow-up PR for that.